Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -841,3 +841,63 @@ label mas_wrs_twitch:
     if not wrs_success:
         $ mas_unlockFailedWRS('mas_wrs_twitch')
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_totono",
+            category=['you', 'and', 'me', 'and', 'her:', 'a', 'love', 'story'],
```

Comment:
So this is going to need to be a lot more specific, or sites with any of these words in their handles will trigger this. Which would basically be most websites